docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md

```diff
@@ -6,26 +6,26 @@ ms.assetid: 69a21511-0871-4c41-9a53-93110e84d7fd
 # Troubleshooting the Getting Started Tutorial
 This topic lists the most common problems encountered when working through the Getting Started Tutorial and how to resolve them.  
   
-1.  [I am unable to find the project files on my hard drive.](../../../docs/framework/wcf/troubleshooting-the-getting-started-tutorial.md#BKMK_q1)  
+1.  [I am unable to find the project files on my hard drive.](#BKMK_q1)  
```

Comment:
not related to your change but this topic looks pretty bad with this layout. The section names look ridiculously long. Wondering if removing this menu from the top and just listing the different problems on a bulleted list could make this look a bit better